src/cdk-experimental/column-resize/resizable.ts

```diff
@@ -114,20 +116,23 @@ export abstract class Resizable<HandleComponent extends ResizeOverlayHandle>
     // of two table cells and is also useful for displaying a resize thumb
     // over both cells and extending it down the table as needed.
 
+    const isLtr = this.directionality.value === 'ltr';
     const positionStrategy = this.overlay.position()
         .flexibleConnectedTo(this.elementRef.nativeElement!)
         .withFlexibleDimensions(false)
         .withGrowAfterOpen(false)
         .withPush(false)
+        .withDefaultOffsetX(isLtr ? 0 : 1)
         .withPositions([{
-          originX: 'end',
+          originX: isLtr ? 'end' : 'start',
           originY: 'top',
           overlayX: 'center',
           overlayY: 'top',
         }]);
 
     return this.overlay.create({
-      direction: this.directionality,
+      // Always position the overlay based on left-indexed coordinates.
+      direction: new Directionality(),
```

Comment:
`Directionality` is meant to be injected; could this be passed in from somewhere else?

Reply:
It's intentionally not using the injected one to force the overlay to use LTR mode (where it sets style.left rather than style.right. The logic for positioning it (both in actual LTR and RTL contexts) is much simpler this way.

Reply:
You can also pass in `ltr` as a string.

Reply:
Much nicer, thanks!